public/locales/en.json

```diff
@@ -49,7 +49,7 @@
   "addPoolNameAndSymbol": "Add pool name and symbol",
   "addPoolNameExample": "Pool name example: alETH pool, MIM Metapool.",
   "addPoolSymbolExample": "Pool Symbol example: saddle-alETH, saddle-MIM-3pool.",
-  "amplificationParameter": "Amplification parameter - controls tolerance for imbalance within the pool's assets. Higher A means lower tolerance for imbalance.",
+  "amplificationParameter": "Amplification parameter - controls tolerance for imbalance within the pool's assets. Higher A means a more stable and less change in price.",
```

Comment:
Should it be `more stability and less change in price`?

Reply:
Here's the tooltip text we're already using in the pool overview: `Amplification coefficient. Higher values widen the range of low-slippages swaps, while lower values help keep the pool's composition balanced.`

Maybe like `Higher A means lower slippage but is only appropriate for tightly pegged assets to avoid impermanent loss.` 🤷‍♂️

Reply:
That's a good way to put it. It's also an example that is nice.